website/src/components/Edit/SequencesForm.spec.tsx

```diff
@@ -112,16 +113,22 @@ describe('SequencesForm', () => {
 
         let secondKey;
         {
-            editableSequences = editableSequences.update(firstKey, 'ATCG', 'Segment 1', 'subId_Segment1');
+            editableSequences = editableSequences.update(firstKey, 'ATCG', 'Segment 1', 'subId_Segment1 description');
```

Comment:
Next time you should use a constant here for the magic strings `subId_Segment1` etc this would make the changes just be in one place (adding ` description` there rather than in 10 places.